service/waterfall.go

```diff
@@ -535,6 +544,7 @@ func countOnPreviousPage(skip int, numVersionElements int,
 // Http handler for the waterfall page
 func (uis *UIServer) waterfallPage(w http.ResponseWriter, r *http.Request) {
 	projCtx := MustHaveProjectContext(r)
+	isSuperUser := auth.IsSuperUser(uis.Settings.SuperUsers, gimlet.GetUser(r.Context()))
```

Comment:
I hate to say this, but in middleware.go/LoadProjectContext, we basically already make this call, so really we should just make the projContext persist "isSuperUser" as a bool (it already does "is admin", so this isn't really a stretch,) rather than duplicating this check here.